### PR TITLE
Travis CI -> CircleCI 2.0

### PR DESCRIPTION
**Pull request by glasnt**

### CODE REVIEW

**Comment by vxsx** [comment]
Hmm, it looks like the frontend tests are missing, it's my bad, they should've been reenables in this PR: https://github.com/aldryn/aldryn-events/pull/246/files#diff-354f30a63fb0907d4ad57269548329e3R40

**Comment by vxsx** [comment]
Could you add them? or at least show me how to add them?

**Comment by glasnt** [comment]
@vxsx which dj/cms version do you want FE tests on? `py35-dj19-cms33-fe` was dropped in #253

**Comment by vxsx** [comment]
py{doesn't matter}-dj{18,19}-cms34-fe

**Comment by glasnt** [comment]
@vxsx I took the `djangocms-casper-helpers` change from another repo; FE should now run green

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+ARG PYTHON_VERSION=3.6
```
Comment (czpython) [inline Dockerfile]: move to `.circleci/Dockerfile`

Comment (glasnt) [reply Dockerfile]: Useful change! Will apply this across all addons, after this is approved/merged

**Comment by glasnt** [comment]
@vxsx https://circleci.com/gh/aldryn/aldryn-events/97 and https://circleci.com/gh/aldryn/aldryn-events/88 are still having FE based issues. Any ideas?

**Comment by vxsx** [comment]
@glasnt that change is a breaking one, it would expect the cms 3.5 

but there seems to be a real issue there: `AttributeError: 'EventConfigAdmin' object has no attribute 'declared_fieldsets'` 

i'll have to test locally

**Comment by vxsx** [comment]
This is related to django-appdata - fixed on master

**Pull request merged**